cpp/include/cudf/functions.h

```diff
@@ -730,3 +730,20 @@ gdf_error gdf_from_dlpack(gdf_column** columns,
 gdf_error gdf_to_dlpack(DLManagedTensor_ *tensor,
                         gdf_column const * const * columns,
                         gdf_size_type num_columns);
+
+/* ----------------------------------------------------------------------------*/
+/** 
+ * @brief Checks if the rows of a `table` are sorted in a lexicographical order.
+ * 
+ * @param[in] input_table        whose rows needs to be validated 
+ * @param[in] descending         vector representing the relation between columns
```

Comment:
```suggestion
 * @param[in] descending         vector that specifies the expected ordering of each input column
```

Reply:
Question: should you accept  an empty vector to specify the default of all ascending order?

Reply:
That would be a good addition to the functionality.